src/test/resources/smallerImage.raw

```diff
@@ -0,0 +1 @@
(.R>/;(&(* 7-)600623'%%/,,af�lot������y{|���������z{t���xwp_]QroeZUGjaOa\OPH5NF6E=,>5&WG24+&+fD7�eSN9,#&.!<)oQ@A, *2#1#*"4.+622100SXk���}��\^W������wxs���������ZZN`[MhdZPL<`XH]TEC;(RJ7C;):1 ^J6.(#![;*�dQN4%-+* D1%�eVsSDQ8+. -"-%6.'*%"420GHXw}�������opj������uwufi_���TSEXS@kgZ]VHJB.OE1KA-8091 80_J64,7-+#( C- M5(.+R=1�tc}`OX>10"*!+"-$,'!)'$9:Ims�tvt������z}x|~|���������kh_bZPB<0VM<YP<>8#;4 F=*5-71[K54,6-'$3+-#-&?._J;K6'oVJC0"0!/!,$)$'&!())nr����kll������}�~���~�}���KD6h`Q\UFB:+G>)OF0<4#<4G8!I<&UG27/3-&"%#,)(%,(;6)CD?9:5;:/\YU50&)#1*3.# SS\���tuv������������|~~���QOC86&JA,PG0C:'A4#A8#>7$80N?+OA.0).'($ 228;963-:8+:93AB=]^`RUYB?4?:(FE6FD<TSN#!WWV������������������������ZSDVP?KD.C='ND/E;)70;070QA+F:%.)4,)' DBLkeq;7'B@9kloUZ]LPNejlXZZVWRuvvegf���.-.qsr�������rtp������������<9(E<%ND,B>*60;4 ?6"6..*N>&K<%1)0,>2"VB5VC?;76>=2JKL_adx~�dhk}��UXVnnp���wxy��� !GFG������vyt���}}���}{������LI760!6.<4"B=(4/4.=08/E6R@)3+3-($3)4*%88>HJJIOQ~��ejn_ce}��\^[���}���wxvbdd���������z|y������klg������@9&JB/D:-7-?5B: 7/1(3-90N>+5/ 73&*) '#=<>Z]^PVX���pu{���x{opr���cfd���PPO"~�����rvq���sur��������}������?8&B8"OD-NC,813,820+0*;5#-&<4$;8(,+ 3+#VPVRTWqrwwz�`ed���dih������������E>9 !$'#������������||����}���fe`���PG2D>+91D<(6.!4+C:#3+5/4.0*,'  #((+\\l=?>_`a���������z~����fjf~�����<80 &($���lpm��������~��{~���oni���G?'72"2,I=)6,82 2-1*<1/)4. ! !# ,3>?DVORaRVVmpsdee���nrq���mpozzwqsr}�0/)5.'aTY���������mnm���nrm������lnkmokJ@*90 <4"6.3,<1"/*6.9.2-0)" (("+,++,0JQ^EILx|}quu������txw���qqr���imn|~y//)0( I?@PSb������hke������\_[gic������5.C9(701*6/4-,&>7%50@<(=8+1*)($&45+CDDWYZ���vwy���~�}������������bec���DDE! ;=J���������qtq��~������non���>3";5#5.3,93!5094$ID4:8*KJ@GFA?=0@>5;;6bc_WXXrur���������|}������������ZZQOQJ./)#"!15AT[vbl}VYVsvn������eicy}x������H8!A9(71!<4$74'1/!DB4KJACE:ikhOOMedc[[Zijjy{zhjh���UWS�����������ehe������NL@;5'&"""!25CLRg���������VYQ������x|}prp���N851#>8(G@1;;264*ZVPTUSYZTbccZ[XssrUUQ���hjgac]���}��������rwr���xywED:GC6OJ77/&"('*+-38D{��lni���������y{x��������\<D<.A>.II@A@=JJDeegKKFcd`RRIiigz}�^\Uuus\`^fgc���Z^__bY���klhLLAWWMjj_ED@52)7/:050-/117@������moj}}w���������eha���lBJ:)YVGbc_PPJdd`HHDNOFhikKI>{zt���\ZWgif���KNGvxo���bb^<<1omcjjdMMEFF>QPMmkfOH9B;)<3!2. ikppsr��y������tvuoqi���������uHN8!>9-UUNKLFZ[X>>6pnf���qrqIG>jia��~MOK\^[�HIFEC9ie`qpk_]YKNDijf���llkttsMKAgaUb_Tcc]|{v������ihdfga������w{~WXTyzv{P"ZD(BA8KH:_^XjhgXZXGE;VUI����CF=UVI{zztvw9</_`WffbhihRSItuo������ehiZZS������WWRomb������jpk[^R������OSKpsn����������W'hDPIAVTMMKGJJ@\^Y���GHFJF?yww���NNJB?5kgbdba<<1BB5loe���XZWXXU������wzz]_[������x{xWWQwws���lnn^b[z}u������dea�������^+zQ#P?'``XijjVXX53'POEtsodgjEGBRVLomhdddRRIfe]nngkjhgigQSHfjd������_b\������efffhdvxu���sy{`a[������a`\������bd\���tvs
```

Comment:
I cannot see the image, but are you sure they have a good license?

Reply:
I've droped this raw files altogether, and switched to reusing images we already have in the repo.